src/echo/subscriber.js

```diff
@@ -445,6 +445,28 @@ class Subscriber extends EventEmitter {
 		return null;
 	}
 
+	/**
```

Comment:
remove

Reply:
I don't understand, should i delete this  /**  thing? If I do that then a disaster will happened with this code)